packages/orn-locator/src/ornPatterns.ts

```diff
@@ -0,0 +1,7 @@
+
+export const patterns = {
+  ['https\\://openstax.org/orn/content\\:book/:id']: ({id}: {id: string}) => import('./resolvers/books').then(mod => mod.bookDetail(id)),
+  ['https\\://openstax.org/orn/content\\:collection/:bookId\\::collectionId']: (params: any) => import('./resolvers/books').then(mod => mod.collection(params)),
+  ['https\\://openstax.org/orn/content\\:page/:bookId\\::pageId']: (params: any) => import('./resolvers/books').then(mod => mod.page(params)),
+  ['https\\://openstax.org/orn/content\\:element/:bookId\\::pageId\\::elementId']: (params: any) => import('./resolvers/books').then(mod => mod.element(params)),
```

Comment:
I think the idea we had (maybe just me so not sure) was to allow a type hierarchy. Maybe `content` is the wrong word here? And instead it could be something like:

```py
https://openstax.org/orn/book/{book_uuid}
https://openstax.org/orn/book:page/{book_uuid}:{page_uuid}
https://openstax.org/orn/book:page:element/{book_uuid}:{page_uuid}:{element_id}
```